src/current/v23.2/column-level-encryption.md

```diff
@@ -0,0 +1,331 @@
+                        2,
+                        '0'
+                    )
+                || '-'
+                || lpad(
+                        (10000 * random())::INT8::STRING,
+                        4,
+                        '0'
+                    )
+                    AS ssn
+            FROM
+                ROWS FROM (generate_series(1, 1.0E+04))
+        )
+SELECT
+    ssn
+FROM
+    a
+~~~
+
+Using both `encrypt` and `decrypt`, the following statement generally ran in 80-100ms:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+WITH
+    a
+        AS (
+            SELECT
+                lpad(
+                    (1000 * random())::INT8::STRING,
+                    3,
+                    '0'
+                )
+                || '-'
+                || lpad(
+                        (100 * random())::INT8::STRING,
+                        2,
+                        '0'
+                    )
+                || '-'
+                || lpad(
+                        (10000 * random())::INT8::STRING,
+                        4,
+                        '0'
+                    )
+                    AS ssn
+            FROM
+                ROWS FROM (generate_series(1, 1.0E+04))
+        )
+SELECT
+    convert_from(
+        decrypt(
+            encrypt(
+                ssn::BYTES,
+                e'\\xd54a43d2a4caf8d3fbe4e4f711b39d4a0fedf26ac0dcdfb0811c2078a6a9cd147e77da38e35e14cacfc79c7e11a052c4bc9449e1d6fa280dcdc45bb4004f1648',
+                'aes'
+            ),
+            e'\\xd54a43d2a4caf8d3fbe4e4f711b39d4a0fedf26ac0dcdfb0811c2078a6a9cd147e77da38e35e14cacfc79c7e11a052c4bc9449e1d6fa280dcdc45bb4004f1648',
+            'aes'
+        ),
+        'UTF8'
+    )
+FROM
+    a
+~~~
+
+With `encrypt` only, the following statement generally ran in 80-100ms:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+WITH
+    a
+        AS (
+            SELECT
+                lpad(
+                    (1000 * random())::INT8::STRING,
+                    3,
+                    '0'
+                )
+                || '-'
+                || lpad(
+                        (100 * random())::INT8::STRING,
+                        2,
+                        '0'
+                    )
+                || '-'
+                || lpad(
+                        (10000 * random())::INT8::STRING,
+                        4,
+                        '0'
+                    )
+                    AS ssn
+            FROM
+                ROWS FROM (generate_series(1, 1.0E+04))
+        )
+SELECT
+    encrypt(
+        ssn::BYTES,
+        e'\\xd54a43d2a4caf8d3fbe4e4f711b39d4a0fedf26ac0dcdfb0811c2078a6a9cd147e77da38e35e14cacfc79c7e11a052c4bc9449e1d6fa280dcdc45bb4004f1648',
+        'aes'
+    )
+FROM
+    a
+~~~
+
+{{site.data.alerts.callout_info}}
+It is important to benchmark these built-in functions on your particular CockroachDB setup to establish the performance implications for your workloads.
```

Comment:
Let's add some examples:
```
It is important to benchmark these built-in functions on your particular CockroachDB setup to establish the performance implications for your workloads. Since performance can vary depending on your hardware (CPU type), load on the cluster etc.
```

Reply:
Sounds good, added basically your suggested text:

> This is necessary because performance can vary depending on your hardware (CPU type), the typical amount of load on the cluster, etc.